marshal/cluster.go

```diff
@@ -173,14 +183,14 @@ func (c *cluster) refreshMetadata() error {
 }
 
 // getOffsetCoordinator returns a kafka.OffsetCoordinator for a specific group.
-func (c *cluster) getOffsetCoordinator(groupID string) (kafka.OffsetCoordinator, error) {
+func (c *KafkaCluster) getOffsetCoordinator(groupID string) (kafka.OffsetCoordinator, error) {
 	return c.broker.OffsetCoordinator(
```

Comment:
If `broker` is closed, are we sure this returns an error instead of panicking or something? (Concerned about calls to `NewMarshaller` on a terminated `KafkaCluster`)